pkg/cloudprovider/huaweicloud/nat_connections.go

```diff
@@ -297,7 +297,7 @@ func (nat *NATClient) CreateDNATRule(dnatRuleConf *DNATRule) (*DNATRule, error)
 }
 
 func (nat *NATClient) DeleteDNATRule(dnatRuleId string) error {
-	url := "/v2.0/dnat_rules/" + dnatRuleId
+	url := "/v2/dnat_rules/" + dnatRuleId
```

Comment:
we should use the following API:
`DELETE /v2/{project_id}/nat_gateways/{nat_gateway_id}/dnat_rules/{dnat_rule_id}`